project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
```

Comment:
Why are we special casing PE here? Why don't we just bump the revision for FOSS also? This particular change would have zero CI pipeline testing coverage for starters today, I'd at least feel more comfortable if FOSS was being tested with the new rev.